polyfill/test/datetime.mjs

```diff
@@ -288,6 +291,12 @@ describe('DateTime', () => {
     it('DateTime.from({ year: 1976, month: 11, day: 18 }) == 1976-11-18T00:00', () => equal(`${DateTime.from({ year: 1976, month: 11, day: 18 })}`, '1976-11-18T00:00'));
     it('DateTime.from({ year: 1976, month: 11, day: 18, millisecond: 123 }) == 1976-11-18T00:00:00.123', () => equal(`${DateTime.from({ year: 1976, month: 11, day: 18, millisecond: 123 })}`, '1976-11-18T00:00:00.123'));
     it('DateTime.from({}) throws', () => throws(() => DateTime.from({}), RangeError));
+    it('DateTime.from(ISO string leap second) is constrained', () => {
+      equal(`${DateTime.from('2016-12-31T23:59:60')}`, '2016-12-31T23:59:59');
+    });
+    it('DateTime.from(property bag leap second) throws', () => {
+      throws(() => DateTime.from({ year: 2016, month: 12, day: 31, hour: 23, minute: 59, second: 60 }), RangeError);
```

Comment:
I thought the default behavior for property bag was to constrain

Reply:
The default behaviour for the constructor is to constrain, that of from() is to reject. But after #260 is resolved, it'll be the other way around.